tests/Old/CommonExtractorTest.php

```diff
@@ -308,15 +308,9 @@ public function testTestConnectionFailure(): void
         $config['action'] = 'testConnection';
         $config['parameters']['tables'] = [];
         $config['parameters']['db']['#password'] = 'bullshit';
-        $app = $this->getApp($config);
-        $exceptionThrown = false;
-        try {
-            $app->run();
-        } catch (\Keboola\DbExtractor\Exception\UserException $e) {
-            $exceptionThrown = true;
-        }
 
-        $this->assertTrue($exceptionThrown);
+        $this->expectException(UserException::class);
```

Comment:
tady prosim pridat jeste test na message (jinak to muze byt falesny success)

Reply:
Upraveno! Projel jsem i zbyle expectException a eliminoval false positive co to slo.